### DISCUSSION TITLE
I have been blocked from the flutter organization github repo

### DISCUSSION THREAD

**Discussion by cp-pratik-k** [community/community#69608] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have been blocked from the Flutter organization GitHub repo. Maybe it's because I created an issue in the Flutter repo. Maybe I forgot some community guidelines. I want to delete this issue to follow the guidelines, but I am unable to do so because of the block.

What i can do now

**Comment by vishalkrishna8** [comment, marked as answer]
Hi @cp-pratik-k , 
First contact to the repository administrators or maintainers.
You can send an email or message through other communication channels to explain the situation and express your willingness to follow the guidelines, and request assistance in resolving the issue.
If you are unable to resolve the issue through communication with the repository administrators and you believe it's essential to continue contributing, you might consider creating a new GitHub account.
Let me know if this was helpful to you.
